pkg/kubelet/cm/dra/plugin/client_test.go

```diff
@@ -85,11 +85,12 @@ func TestGRPCConnIsReused(t *testing.T) {
 	wg := sync.WaitGroup{}
 	m := sync.Mutex{}
 
-	plugin := &Plugin{
-		endpoint: addr,
+	p := &plugin{
+		endpoint:        addr,
+		fallbackVersion: "v1alpha3",
```

Comment:
nit: field name should be changed. v1alpha3 is not a fallback version.

Reply:
done in a57da3914ab